shap/explainers/_permutation.py

```diff
@@ -48,7 +49,7 @@ def explain_row(self, *row_args, max_evals, main_effects, error_bounds, batch_si
 
         # by default we run 10 permutations forward and backward
         if max_evals == "auto":
-            max_evals = 10 * 2 * len(fm)
+            max_evals = 10 * 2 * len(fm) + 1
```

Comment:
what was the reasoning behind the +1 here?

Reply:
I believe this is to ensure that even if len(fm) is 0, we would at least evaluate once. However, I'm not absolutely sure about it.